app/views/layouts/_navbar.haml

```diff
@@ -1,12 +1,34 @@
-%nav.navbar.navbar-expand-lg.mb-2.bg-light{aria: {label: 'Main'}}
-  - if @current_user
-    = navbar_link 'Passengers', passengers_path
-    = navbar_link 'Add New Passenger', new_passenger_path
-    = navbar_link 'Dispatch Log', log_index_path
-    = navbar_link 'Archived Passengers', archived_passengers_path
-  - if @current_user&.admin?
-    = navbar_link 'Manage Users', users_path
-    = navbar_link 'Manage Mobility Devices', mobility_devices_path
-  = navbar_link 'Site Feedback', feedback_index_path
-  = navbar_link 'Brochure', brochure_passengers_path
-  = link_to 'Logout', destroy_session_path, class: 'ml-auto mr-2 nav-link btn btn-secondary'
+%nav.navbar.navbar-expand-lg.navbar-light.bg-light{aria: {label: 'Main'}}
+  %ul.navbar-nav.mr-auto
+    - if @current_user
+      %li.nav-item.dropdown
+        = link_to '#', class: 'nav-link dropdown-toggle',
+          id: 'passenger-dropdown',
+          data: {toggle: 'dropdown'},
+          aria: {haspopup: 'true', expanded: 'false'} do
+          Passengers
+          - unless Passenger.pending.count.zero?
+            %i.fas.fa-user-cog{aria_label: 'Pending passengers present'}
```

Comment:
The "cog" in the icon says to me "settings" or "preferences". For the concept of "pending", I think I prefer the clock (implying that they're waiting) or perhaps the "+"?

Also, I would tie it to the internal badge by color (add `.text-warning`)

Reply:
I like the clock better. The plus doesn't indicate, to me, something that you could make disappear by doing some actions.